src/main/java/seedu/placebook/model/person/Phone.java

```diff
@@ -9,28 +9,30 @@
  */
 public class Phone {
 
+    public static final int MAXIMUM_PHONE_NUMBER_LENGTH = 20;
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Phone numbers should only contain numbers, and it should be at least 3 digits long";
+            "Phone numbers should only contain numbers, it should be at least 3 digits long"
+                    + " and at most " + MAXIMUM_PHONE_NUMBER_LENGTH + " digits long";
     public static final String VALIDATION_REGEX = "\\d{3,}";
```

Comment:
I think changing this to `"\\d{3,20}";` will do the job

Reply:
Oh ya changing the validation regex is a better way to do this. Thank you!